src/index.html

```diff
@@ -3,11 +3,394 @@
+
+          <div class="events__card-inf">
+            <div class="events__inf events__inf-last">
+              <div class="events__type">ЛЕКЦІЯ</div>
+              <div class="events__date">26/08/2019 - 11:00</div>
+            </div>
+
+            <h3 class="events__subtitle events__subtitle-last">Від класицизму до <br>романтизму</h3>
+          </div>
+        </div>
+      </section>
+
+      <section class="galery" id="galery">
+        <div class="container">
+          <h2 class="galery__title subtitle">Галерея</h2>
+
+          <div class="galery__slaider slaider">
+            <div class="slaider__item">
+              <picture class="slaider__images-1">
+                <source
+                  media="(min-width: 1280px)"
+                  srcset="images/galery/1-desctop.jpg"
+                >
+
+                <source
+                  media="(min-width: 640px)"
+                  srcset="images/galery/1-tablet.jpg"
+                >
+
+                <img
+                  src="images/galery/1-mobile.jpg"
+                  alt="girl"
+                >
+              </picture>
+
+              <picture class="slaider__images-2">
+                <img
+                  src="images/galery/2-desctop.jpg"
+                  alt="house"
+                >
+              </picture>
+
+              <picture class="slaider__images-3">
+                <img
+                  src="images/galery/2-tablet.jpg"
+                  alt="duck"
+                >
+              </picture>
+            </div>
+            <div class="slaider__item">
+              <picture class="slaider__images-4">
+                <img
+                  src="images/galery/3-desctop.jpg"
+                  alt="chair"
+                >
+              </picture>
+
+              <picture class="slaider__images-5">
+                <img
+                  src="images/galery/4-desctop.jpg"
+                  alt="sea"
+                >
+              </picture>
+            </div>
+
+            <div class="slaider__dots">
+              <div class="slaider__dot--dark"></div>
+              <div class="slaider__dot--light"></div>
+              <div class="slaider__dot--light"></div>
+              <div class="slaider__dot--light"></div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="subscription" id="subscription">
+        <div class="container">
+          <div class="subscription__body">
+            <h2 class="subscription__title subtitle">Підписка</h2>
+
+            <div class="subscription__text">
+              Бути в курсі всіх актуальних подій та останніх новин.
+            </div>
+
+            <div class="subscription__button">
+              <form
+                class="subscription__form"
+                action="#"
+                name="form"
+                method="post"
+                id="form"
+                onsubmit="return signup();"
```

Comment:
it will work better with such handling: [checklist](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md) 12 (the checklist from another landing page task)